tests/collectors/hra/test_collector.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import mock
+import pytest
+import datetime
+import requests
+from collections import defaultdict
+from collectors.hra.collector import collect, _make_request_url
+
+
+class TestHRACollector(object):
+    def test_collect(self, conf, conn):
+        if conf['HRA_ENV'] == 'staging':
+            with pytest.raises(SystemExit):
+                collect(conf, conn, '2015-01-01', '2015-01-01')
+            assert conn['warehouse']['hra'].count() > 10
+
+    def test_make_request_url(self):
+        date_from = datetime.date(2015, 01, 01)
```

Comment:
Python treats numbers starting with `0` as octal, like:

```python
>>> print(011)
9
```

So this should be `datetime.date(2015, 1, 1)`